src/server/service/kv.rs

```diff
@@ -461,6 +465,7 @@ impl<E: Engine, L: LockManager, F: KvFormat> Tikv for Service<E, L, F> {
             sink.success(resp).await?;
             GRPC_MSG_HISTOGRAM_STATIC
                 .kv_flashback_to_version
+                .unknown
```

Comment:
maybe need to unify `.get(ResourcePriority::unknown)` with `.unknown`
```suggestion
		.get(ResourcePriority::unknown)
```

Reply:
fixed